docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,250 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/beyla.ebpf/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/beyla.ebpf/
+description: Learn about beyla.ebpf
+title: beyla.ebpf
+---
+
+# beyla.ebpf
+
+The `beyla.ebpf` component is used a wrapper of [Grafana Beyla][] which collects metrics and traces from eBPF programs running on the host. The component can be configured to collect data from a specific port or executable name, and other criteria from Kubernetes metadata. The component can  be scraped by Prometheus and can send traces to an OpenTelemetry Collector.
+
+{{< admonition type="note" >}}
+To run this component, the Agent requires administrative (`sudo`) privileges, or at least it needs to be granted the `CAP_SYS_ADMIN` capability.
```

Comment:
We also need CAP_SYS_PTRACE and when running in k8s we should mention that app armour must be disabled for the deployment, e.g:

```
container.apparmor.security.beta.kubernetes.io/beyla: "unconfined" 
```